HPXMLtoOpenStudio/resources/geometry.rb

```diff
@@ -526,16 +526,20 @@ def self.apply_occupants(model, hpxml, num_occ, cfa, space, schedules_file)
     occ_lost = 1 - occ_lat - occ_conv - occ_rad
 
     # Create schedule
+    people_sch = false
     if not schedules_file.nil?
       people_sch = schedules_file.create_schedule_file(col_name: 'occupants')
-    else
+    end
+    if not people_sch
       weekday_sch = hpxml.building_occupancy.weekday_fractions.split(',').map(&:to_f)
       weekday_sch = weekday_sch.map { |v| v / weekday_sch.max }.join(',')
       weekend_sch = hpxml.building_occupancy.weekend_fractions.split(',').map(&:to_f)
       weekend_sch = weekend_sch.map { |v| v / weekend_sch.max }.join(',')
       monthly_sch = hpxml.building_occupancy.monthly_multipliers
       people_sch = MonthWeekdayWeekendSchedule.new(model, Constants.ObjectNameOccupants + ' schedule', weekday_sch, weekend_sch, monthly_sch, Constants.ScheduleTypeLimitsFraction)
       people_sch = people_sch.schedule
+    elsif !hpxml.building_occupancy.weekday_fractions.nil? && !hpxml.building_occupancy.weekend_fractions.nil? && !hpxml.building_occupancy.monthly_multipliers.nil?
+      puts "Warning: Using 'occupants' schedule even though weekday/weekend fractions and monthly multipliers are provided."
```

Comment:
You need to use `runner.registerWarning`, not `puts`, otherwise the warning won't show up as appropriately. This also suggests to me that you don't have a test to check for the warning because the test would have failed. Can you add a `test_measure_warning_messages` in test_validation.rb?

I also suggest making the warnings more nuanced. How about:

```
  else
      runner.registerWarning("Both 'occupants' schedule file and weekday fractions provided; the latter will be ignore.") if !hpxml.building_occupancy.weekday_fractions.nil?
      runner.registerWarning("Both 'occupants' schedule file and weekend fractions provided; the latter will be ignore.") if !hpxml.building_occupancy.weekend_fractions.nil?
      runner.registerWarning("Both 'occupants' schedule file and monthly multipliers provided; the latter will be ignore.") if !hpxml.building_occupancy.monthly_multipliers.nil?
```